### DISCUSSION TITLE
Captions for JourneyDB during the training process

### DISCUSSION THREAD

**Discussion by jiangzeyinzi** [PixArt-alpha/PixArt-alpha#99] [Q&A] [answered]
Thank you for your excellent work. 
I would like to know if the captions used for training in the 4M JourneyDB of the HQ dataset originate from the LLaVA annotation process described in the paper, are taken from the original dataset 'prompt', or are the 'Caption' from 'Task 2'.

**Comment by lawrence-cj** [comment, marked as answer]
the original prompt
